### PR TITLE
adding SDK guides for readme.io for readme-sync

### PR DESCRIPTION
**Pull request by fscelliott**
## Summary
- adding SDK guides in readme-markdown (currently hosted at: https://dash.readme.io/project/full-stack/v3.1/settings) so JC's readme-sync travis proof of concept tool works.

The "why", or other context.
docs-as-code approach

## Issues
no formal jira corresponds; see #devx-readme-sync slack channel

### CODE REVIEW

**Comment by fscelliott** [comment]
@juancarlostong sorry, I didn't dig into travis, but is it likely this is failing for the same spell checker reason?

**Comment by juancarlostong** [comment]
yes thats right, ill fix it in your branch and bring your branch up to date with master so you wont have problems merging. remember to git pull your local branch

**Comment by mikeng13** [comment]
@fscelliott or @juancarlostong please update the PR summary and make sure to fill in that section.

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+slug: "csharp-sdk"
+hidden: false
+metadata: 
+  title: "Optimizely C# SDK"
+createdAt: {}
+updatedAt: "2020-01-22T23:31:33.332Z"
+---
+This reference guide describes how to use the Optimizely C# SDK. Refer to the left hand navigation for all sections in this reference guide.
+
+The Optimizely C# SDK is currently at version 3.2.0.
```
Comment (mikeng13) [inline docs/index.md:12]: Please update this to the latest version, which is 3.4.1

Comment (fscelliott) [reply docs/index.md]: done

**Pull request merged**